app/uk/gov/hmrc/hmrcemailrenderer/templates/fhdds/fhddsBetaSubmissionConfirmation.scala.html

```diff
@@ -0,0 +1,49 @@
+@(params: Map[String, Any])
+@uk.gov.hmrc.hmrcemailrenderer.templates.helpers.html.template_main(params, "This is FHDDS Beta submission email."){
+<p style="margin: 0 0 30px; font-size: 19px;">
+    Dear Customer: <br><br>
+
```

Comment:
Paragraphs can be encased in +< p style="margin: 0 0 30px; font-size: 19px;"> < /p>  tags then < br> won't look messy